geomstats/geometry/invariant_metric.py

```diff
@@ -14,6 +14,15 @@ class InvariantMetric(RiemannianMetric):
 
     Points are parameterized by the Riemannian logarithm
     for the canonical left-invariant metric.
+
+    Parameters
+    ----------
+    group : LieGroup
+        The group to equip with the invariant metric
+    inner_product_mat_at_identity : array-like, [dimension, dimension]
```

Comment:
shape=

Reply:
When we replace `dimension` by `dim` everwhere we won't have the lines too long problem: https://github.com/geomstats/geomstats/issues/376